templates/comments/list.html

```diff
@@ -66,39 +66,32 @@ <h2><i style="padding-right: 0.3em" class="fa fa-comments"></i>{{ _('Comments')
                                 {% else %}
                                     <span class="comment-edits"></span>
                                 {% endif %}
-                                    <a href="#comment-{{ node.id }}" title="{{ _('Link') }}" class="comment-link">
-                                    <i class="fa fa-link fa-fw"></i>
-                                </a>
+                                    <a href="#comment-{{ node.id }}" title="{{ _('Link') }}"
+                                       class="comment-link"><i class="fa fa-link fa-fw"></i></a>
```

Comment:
I think `</a>` should be on its own line.

Reply:
`</a>` appears multiple times, and moving each `</a>` into a new line hurts readability